Tests/MoyaTests/TestPlugin.swift

```diff
@@ -13,6 +13,12 @@ final class TestingPlugin: PluginType {
         return request
     }
 
+    func prepare(_ request: URLRequest, multipartBody: [MultipartFormData], target: TargetType) -> (URLRequest, [MultipartFormData]) {
+        var request = request
+        request.addValue("yes", forHTTPHeaderField: "prepared-multipart-body")
```

Comment:
Shouldn't this be asserted at some point of the test? :)

Reply:
Do you mean I need to write a complete upload file test? :)

Reply:
I'm really not sure, I just think if you add this header in a test you should assert that it on the request at somepoint ...

Reply:
Hi @LucianoPAlmeida Since this PR has been closed, no further updates are needed. Thanks for your comments.